app/manifest/_base.json

```diff
@@ -59,7 +59,7 @@
     "512": "images/icon-512.png"
   },
   "manifest_version": 2,
-  "name": "__MSG_appName__",
```

Comment:
- I think we should keep name as `"__MSG_appName__"`
- and replace those `"appName"` in messages.json with `"Carbon Wallet"`

Reply:
fixed in b69b23432ac084623bfa33d7caa7153a18cb79fa

Reply:
- Please also fix the `appName` in other locales

Reply:
Will open issue and fix in the future